dll/win32/msvcrt/stubs.c

```diff
@@ -147,7 +148,7 @@ _chvalidchk_l(
     _In_opt_ _locale_t const locale
     )
 {
-#ifdef _DEBUG
+#if 0 // def _DEBUG
```

Comment:
What's happening with those?

Reply:
This is broken and never worked. We don't implement _chvalidator_l.